front/.prettierignore

```diff
@@ -0,0 +1,7 @@
+src/common/ReleaseInformations/json/licenses.json
+src/modules/rollingStock/components/RollingStockSelector/consts/mlgtraffic.json
+src/modules/rollingStock/components/RollingStockSelector/consts/rollingstockDetailDB.json
+src/modules/trainschedule/components/ImportTrainSchedule/rollingstock_opendata2osrd.json
+src/reducers/osrdconf/infra_schema.json
+src/assets/mapstyles/*.json
+src/utils/__tests__/assets/*.json
```

Comment:
I'd add `src/config/openapi-editoast-config.ts`  and `src/config/openapi-gateway-config.ts` too as they are generated

Reply:
Hm, I don't think these are generated? These are hand-written config files to configure the OpenAPI codegen, I believe?

Reply:
yeah `src/config/openapi-editoast-config.ts` and `src/config/openapi-gateway-config.ts` are not generated, 
the one to ignore is `src/common/api/generatedEditoastApi.tssrc/common/api/generatedEditoastApi.ts`

Reply:
Right. Currently `scripts/generate-types.sh` runs `eslint` after generating the files, so it's not strictly necessary to add these files to the ignore list.

Reply:
we don’t really care about the formatting of these files. eslint is run just once at generation.
otherwise they are ignored in the eslint patterns, I would ignore these too in prettier.

Reply:
Right, I copy pasted the wrong thing ^^'